code/script.js

```diff
@@ -34,18 +43,282 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+// Bot greets user
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
+  showMessage("Hi! I can help you find the perfect gift for anyone. <br><br>Please start by telling me your name.", 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
+//The user types name and name is saved
+const handleNameInput = (event) => { 
+  event.preventDefault(); 
+  //name saved for later
+  name = nameInput.value
+  console.log(name)
+  //the users name pops up as a message
```

Comment:
I really like the way you are commenting your code, it makes it much easier to follow, and understand what is going on and where 👊